### PR TITLE
Init model and relationships between them

### PR DESCRIPTION
**Pull request by vjnhat**

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
```
Comment (hieuns) [inline app/models/review.rb]: Thiếu belongs_to address à cậu?

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    t.string   "password"
+    t.string   "name"
+    t.integer  "age"
+    t.string   "job"
+    t.string   "address"
+    t.integer  "phone_number"
+    t.string   "provider"
+    t.string   "uid"
+    t.integer  "role"
+    t.datetime "created_at",   null: false
```
Comment (daothanhcam) [inline db/schema.rb]: thừa dấu cách.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    t.datetime "updated_at",   null: false
```
Comment (daothanhcam) [inline db/schema.rb]: thừa dấu cách.

File: db/migrate/20150708052758_create_users.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :password
+      t.string :name
+      t.integer :age
+      t.string :job
+      t.string :address
+      t.integer :phone_number
```
Comment (ThuBM) [inline db/migrate/20150708052758_create_users.rb]: phone number de string

**Pull request merged**